src/sshd/README.md

```diff
@@ -37,7 +37,7 @@ While the some services automates SSH setup (e.g., when using the GitHub CLI for
     passwd
     ```
 
-3. Forward the SSH port (`22` by default) to your local machine using either the `forwardPorts` property in `devcontainer.json` or the user interface in your tool (e.g., you can press <kbd>F1</kbd> or <kbd>Ctrl/Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> and select **Ports: Focus on Ports View** in VS Code to bring it into focus).
+3. Forward the SSH port (`2222` by default) to your local machine using either the `forwardPorts` property in `devcontainer.json` or the user interface in your tool (e.g., you can press <kbd>F1</kbd> or <kbd>Ctrl/Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> and select **Ports: Focus on Ports View** in VS Code to bring it into focus).
```

Comment:
As mentioned at the bottom of the README, this file is auto-generated from the [devcontainer-feature.json](https://github.com/devcontainers/features/blob/main/src/sshd/devcontainer-feature.json) and NOTES.md. 

Can we make this change in [NOTES.md](https://github.com/devcontainers/features/blob/main/src/sshd/NOTES.md) instead ? as this will be automatically overwritten as soon as this PR is merged 😄  thank you!